Wordle.Api/Wordle.Api/Dtos/MoreWordsDto.cs

```diff
@@ -0,0 +1,9 @@
+﻿using Wordle.Api.Models;
+
+namespace Wordle.Api.Dtos;
+
+public class MoreWordsDto
+{
+    public List<Word>? Words { get; set; }
```

Comment:
```suggestion
    public List<WordDto> Words { get; set; }  = [];
```
I think this is cleaner since it shows that the list was empty which makes more since than having a null value which would involve more handling. I also believe you want this to be your `WordDto`.

Reply:
I agree. I would implement it, but it's past the deadline. Thank you!